ci/images/debian/Dockerfile.dotnet10

```diff
@@ -92,6 +100,12 @@ RUN set -e; \
  && if [ -n "$NUGET_REPO" ]; then \
       dotnet nuget remove source "nexus"; \
       dotnet nuget add source "https://api.nuget.org/v3/index.json" --name "nuget.org"; \
+    fi; \
+    if [ -f /etc/apt/sources.list ]; then \
+      sed -i "s|$DEBIAN_REPO|http://deb.debian.org|g" /etc/apt/sources.list; \
+    fi; \
```

Comment:
I know this 'reverts' `http://security.debian.org` to 'http://deb.debian.org`, but those names point to the same server anyway, so it's not a problem.